packages/concerto-core/lib/introspect/modelfile.js

```diff
@@ -393,13 +393,7 @@ class ModelFile {
                 }
             }
             else {
-                // check whether type is defined in another file
-                const fqn = this.resolveImport(type);
```

Comment:
Here, we get the FQN of the type. For `import { Thing } from org.example` and `import org.example.Thing` this is simple, as all the information to identify that the FQN of `Thing` is `org.example.Thing`.

For wildcard imports `import org.example.*` (which we want to drop), we have to visit the model manager and look through all wildcard imports for the namespace that contains the type we want.

Either way, we end up with the FQN of the type or we throw an error as the type cannot be resolved.